app/views/_includes/candidate-feedback.njk

```diff
@@ -80,7 +80,7 @@
 
   {% if rejectionReasons['actions'] == "Yes" %}
     <h3 class="govuk-heading-s">
-      Something you did
+      Candidate behaviour
```

Comment:
We started with this but we preferred something you did, it was softer, more human and less ambiguous.